src/server/src/routes/users.js

```diff
@@ -10,58 +10,63 @@ dotenv.config();
 
 const router = express.Router();
 
+const mg = mailgun({
```

Comment:
use a more explainable name as everybody knows what "mg" stands for

Reply:
fixed